- case: failure_lash
  disable_cache: false
  main: |
    from returns.result import ResultE, Success, Failure

    def some(arg: int) -> ResultE[int]:
        if arg > 0:
            return Success(arg)
        return Failure(ValueError('test'))

    reveal_type(some(1))  # N: Revealed type is "returns.result.Result[builtins.int, builtins.Exception]"
